### PR TITLE
Fix Bug 1457582 - Replace strings-export and include .properties as part of buildmc

### PR DESCRIPTION
**Pull request by Mardak**
Waiting for Pike's feedback before review.

### CODE REVIEW

**Comment by Mardak** [comment]
`cpx` copies multiple files/directories to a destination without renaming the copied file. That's why I needed to use `cp`

**Comment by Mardak** [comment]
r?@k88hudson Updated PR to use the new destination:

browser/locales/en-US/chrome/browser/activity-stream/newtab.properties

File: yamscripts.yml [hunk 1]
```diff
@@ -24,6 +24,7 @@ scripts:
     # Copy over all of the system-addon directory
     copy: cpx "system-addon/**/{,.}*" $npm_package_config_mc_dir/browser/extensions/activity-stream
     version: node ./bin/update-version.js $npm_package_config_mc_dir/browser/extensions/activity-stream
+    stringsExport: cp locales/en-US/strings.properties $npm_package_config_mc_dir/browser/locales/en-US/chrome/browser/activity-stream/newtab.properties
```
Comment (k88hudson) [inline yamscripts.yml]: `cp` will fail on windows but we're probably using other unixy stuff elsewhere so oh well :P

Comment (k88hudson) [reply yamscripts.yml]: also, do you want to check if this directory already exists? this would fail otherwise

Comment (Mardak) [reply yamscripts.yml]: Yeah for directory existing.. but... except for the first export, the directory and file will already exist. ;) So I decided against an extra mkdir in the common case going forwards. 😛

**Pull request merged**